rest-service/manager_rest/rest/resources_v3_1/deployments.py

```diff
@@ -93,11 +93,30 @@ def put(self, deployment_id, **kwargs):
                 request_dict),
             site_name=_get_site_name(request_dict),
             runtime_only_evaluation=request_dict.get(
-                'runtime_only_evaluation', False)
+                'runtime_only_evaluation', False),
+            labels=_get_labels(request_dict)
         )
         return deployment, 201
 
 
+def _get_labels(request_dict):
+    if 'labels' not in request_dict:
+        return None
```

Comment:
if you `return []` here, then the signature of this function is simpler - just "returns iterable of labels", vs "returns iterable of labels or None", and then you'll be able to drop the `if` inside of resource-manager

Reply:
Nice, thanks!

Reply:
I went over it now and it might be a bit tricky. The default value of labels is `None` in the `resource_manager.create_deployment()` function, so removing the `if` statement might cause issues if there's a code that doesn't pass labels to the function.